pkg/kyma/cmd/test/delete/cmd.go

```diff
@@ -0,0 +1,68 @@
+package del
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/kyma-project/cli/pkg/kyma/cmd/test"
+	"github.com/kyma-project/cli/pkg/kyma/cmd/test/client"
+	"github.com/kyma-project/cli/pkg/kyma/core"
+	"github.com/spf13/cobra"
+)
+
+type command struct {
+	opts *options
+	core.Command
+}
+
+func NewCmd(o *options) *cobra.Command {
+	cmd := command{
+		Command: core.Command{Options: o.Options},
+		opts:    o,
+	}
+
+	cobraCmd := &cobra.Command{
+		Use:     "delete",
+		Short:   "Delete tests on a running Kyma cluster",
+		Long:    `Delete tests on a running Kyma cluster`,
+		RunE:    func(_ *cobra.Command, args []string) error { return cmd.Run(args) },
+		Aliases: []string{"d"},
+	}
```

Comment:
WHen calling --help the "name" argument is not documented ...

Reply:
fixed